heartbeat/monitors/active/icmp/stdloop.go

```diff
@@ -161,7 +161,15 @@ func newICMPLoop() (*stdICMPLoop, error) {
 func (l *stdICMPLoop) runICMPRecv(conn *icmp.PacketConn, proto int) {
 	for {
 		bytes := make([]byte, 512)
+<<<<<<< HEAD
```

Comment:
I just happened to come across this on this backport and wanted to just point it out to make sure it's noticed.